subprojects/shipkit/src/main/groovy/org/shipkit/internal/notes/generator/ReleaseNotesGenerators.java

```diff
@@ -15,11 +15,11 @@
 public class ReleaseNotesGenerators {
 
     /**
-     * @param workDir the working directory where 'git' operations will be executed
-     * @param gitHubApiUrl GitHub API endpoint address, for example: https://api.github.com
-     * @param gitHubRepository GitHub gitHubRepository in format USER|COMPANY/REPO_NAME, for example: mockito/mockito
+     * @param workDir           the working directory where 'git' operations will be executed
+     * @param gitHubApiUrl      GitHub API endpoint address, for example: https://api.github.com
+     * @param gitHubRepository  GitHub gitHubRepository in format USER|COMPANY/REPO_NAME, for example: mockito/mockito
      * @param readOnlyAuthToken read only auth token used to communicate with GitHub
-     * @param ignoredCommit responsible decide if commits should not be included in release notes
+     * @param ignoredCommit     responsible decide if commits should not be included in release notes
```

Comment:
I know that you didn't add this content, but if you have a second you can change it to "responsible for deciding".